s4e-web/src/app/views/map-view/view-manager/layer-picker/layer-picker.component.spec.ts

```diff
@@ -27,7 +27,7 @@ fdescribe('ItemsPickerComponent', () => {
   it('should have star on enabled', () => {
     component.hasFavourite = true;
     component.items = [{cid: 0, caption: '', active: true, favourite: false}];
-    spyOn(component.isFavoriteSelected, 'emit');
+    spyOn(component.isFavouriteSelected, 'emit');
```

Comment:
I haven't noticed it in the previous PR, but those emits should be tested for emitted values as well.
The alternative on spying is just to subscribe to the `EventEmitter`:

```
const p = component.isFavouriteSelected.pipe(take(1)).subscribe().toPromise();
//...
expect(await p).toEqual(<expected_value>)
```

But just checking by `toHaveBeenCalledWith(...)` is also ok.